src/main/webapp/data/userMapData.json

```diff
@@ -1849,6 +1899,7 @@
       "University of New Hampshire",
       "University of New Mexico",
       "University of North Carolina",
+      "University of North Carolina at Wilmington",
```

Comment:
It seems that this both are same university kindly confirm it

Reply:
Yes it is same, I'll keep the last one.